core/tests/puppeteer-acceptance-tests/utilities/user/exploration-editor.ts

```diff
@@ -1655,6 +1656,7 @@ export class ExplorationEditor extends BaseUser {
       );
       return element ? element.textContent : null;
     }, languageCode);
+    this.page.waitForTimeout(30000);
```

Comment:
We wouldn't want to use arbitrary timeouts. 
Please wait for some event/ element.

Reply:
Ah that was accidentally added, removed it